src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -0,0 +1,122 @@
+    /** Collection of categories attached to a policy */
+    policyCategories: OnyxEntry<OnyxTypes.PolicyCategories>;
+};
+
+type WorkspaceCategoriesPageProps = WorkspaceCategoriesOnyxProps & WithWorkspaceAccessProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORIES>;
+
+function WorkspaceCategoriesPage({policyCategories}: WorkspaceCategoriesPageProps) {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const [selectedCategories, setSelectedCategories] = useState<string[]>([]);
+
+    const categoryList = useMemo<PolicyForList[]>(
+        () =>
+            Object.values(policyCategories ?? {}).map((value) => ({
+                value: value.name,
+                text: value.name,
+                keyForList: value.name,
+                isSelected: selectedCategories.includes(value.name),
+                rightElement: (
+                    <View style={styles.flexRow}>
+                        <Text style={[styles.disabledText, styles.alignSelfCenter]}>{value.enabled ? translate('workspace.common.enabled') : translate('workspace.common.disabled')}</Text>
+                        <View style={styles.p1}>
+                            <Icon
+                                src={Expensicons.ArrowRight}
+                                fill={theme.icon}
+                            />
+                        </View>
+                    </View>
+                ),
+            })),
+        [policyCategories, selectedCategories, styles.alignSelfCenter, styles.disabledText, styles.flexRow, styles.p1, theme.icon, translate],
+    );
+
+    const toggleCategory = (category: PolicyForList) => {
+        setSelectedCategories((prev) => {
+            if (prev.includes(category.value)) {
+                return prev.filter((item) => item !== category.value);
+            }
+            return [...prev, category.value];
+        });
+    };
+
+    const toggleAllCategories = () => {
+        const isAllSelected = categoryList.every((category) => category.isSelected);
+        if (isAllSelected) {
+            setSelectedCategories([]);
+        } else {
+            setSelectedCategories(categoryList.map((item) => item.value));
+        }
+    };
+
+    if (Object.keys(policyCategories ?? {}).length === 0) {
+        return <FullScreenLoadingIndicator />;
+    }
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            style={[styles.defaultModalContainer]}
+            testID={WorkspaceCategoriesPage.displayName}
+            shouldShowOfflineIndicatorInWideScreen
+        >
+            <HeaderWithBackButton
+                icon={Illustrations.FolderOpen}
+                title={translate('workspace.common.categories')}
+                shouldShowBackButton={isSmallScreenWidth}
+            />
+            <SelectionList
```

Comment:
Maybe this is know. The headers should be `Name` and `Status`.

<img width="660" alt="image" src="https://github.com/Expensify/App/assets/29673073/e4377dab-31fc-4540-b487-3bc04e352d5c">

But the header in the PR is `Select all`
<img width="914" alt="image" src="https://github.com/Expensify/App/assets/29673073/82af1813-c54a-40ca-9123-aa4f2eddc198">

Reply:
Same as in this [comment](https://github.com/Expensify/App/pull/36324#discussion_r1491195529)